services/ptrarchive.go

```diff
@@ -58,14 +62,45 @@ func (p *PTRArchive) OnDNSRequest(ctx context.Context, req *requests.DNSRequest)
 	p.CheckRateLimit()
 	bus.Publish(requests.SetActiveTopic, p.String())
 
+	v := url.Values{}
+	v.Set("name", "Ava")
```

Comment:
What is this? I don't think you mentioned it in the PR description.

Reply:
Sorry, this must be a mistake. I'll delete it.